.github/workflows/release.yml

```
@@ -7,255 +7,187 @@ on:
           python-version: '3.11'
-      - name: Verify tag matches Cargo.toml
+      - id: version
+        name: Verify tag matches Cargo.toml
         run: |
           set -euo pipefail
           tag="${GITHUB_REF_NAME#v}"
           toml_path="${CARGO_TOML_PATH:-Cargo.toml}"
           cargo_version="$(
-            CARGO_TOML_PATH="$toml_path" \
             python3 - <<'PY' || echo ""
 import os, tomllib
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 35: could not find expected ':'

(syntax-check)